cypher.html.markdown

```diff
@@ -16,29 +16,19 @@ Nodes
 
 **Represents a record in a graph.**
 
-```
-()
-```
+```()```
```

Comment:
@prertik I actually haven't made any changes to this file. I made commit #b57cca8 but then reverted it in #f95558f because of merge conflicts. I'd be happy to fix this up in another PR.